R/lefser.R

```diff
@@ -213,21 +211,21 @@ filterKruskal <- function(relab, group, p.value, method = method) {
 #'     zeller14tn_ra <- relativeAb(zeller14tn)
 #'     
 #'     # (1) Using classes only
-#'     res_group <- lefser(zeller14tn_ra, 
-#'                         groupCol = "study_condition")
+#'     res_class <- lefser(zeller14tn_ra, 
+#'                         classCol = "study_condition")
 #'     # (2) Using classes and sub-classes
-#'     res_block <- lefser(zeller14tn_ra, 
-#'                         groupCol = "study_condition", 
-#'                         blockCol = "age_category")
+#'     res_subclass <- lefser(zeller14tn_ra, 
+#'                         classCol = "study_condition", 
+#'                         subclassCol = "age_category")
 #'                         
 #' @export
 lefser <-
   function(relab,
            kruskal.threshold = 0.05,
            wilcox.threshold = 0.05,
            lda.threshold = 2.0,
-           groupCol = "GROUP",
-           blockCol = NULL,
+           classCol = "CLASS",
+           subclassCol = NULL,
```

Comment:
The lefser function shoul dkeep the `groupCol` and `blockCol` in the function signature. They should go after the `...` similarly to `expr`.